subsys/bluetooth/mesh/Kconfig

```diff
@@ -111,6 +111,14 @@ config BT_MESH_PROXY_FILTER_SIZE
 	  This option specifies how many Proxy Filter entries the local
 	  node supports.
 
+config BT_MESH_ADV_STACK_SIZE
+	int "Mesh advertiser thread stack size"
+	default 1024 if BT_HOST_CRYPTO
+	default 768
+	help
+	  NOTE: This is an advanced setting and should not be changed unless
```

Comment:
In what use-cases is changing the ADV_STACK_SIZE needed?

Reply:
Thanks for review ! We use the zephyr bluetooth stack base on some customized RTOS and hardware(Cortex-A/R), the consumption of thread stack size is not fixed.

Reply:
Is the stack size increase needed because of application callback behavior? Or due to usage from the underlying RTOS/hardware?

I suppose you copied this note from the help text of some of the other BT stack sizes, but if you see BT_RX_STACK_SIZE
where the stack size increase might be needed because callbacks are called from this thread, the help text explains better why this might be needed.

Reply:
Thanks for reminder, In fact, we have different requirements for the basic thread stack size, especially in x86 Simulate, the consumption of the basic stack will be very large.
I know this is not what zephyr expected, but the configurable stack size will be a friendly and flexible option when we want the thread runs in a different environment.

```
$ ps
  PID GROUP PRI POLICY   TYPE    NPX STATE    EVENT     SIGMASK   STACK   USED  FILLED COMMAND
...
    8     7 102 FIFO     Kthread --- Waiting  Semaphore 00000000 008176 002332  28.5%  BT TX  0xf11460f0
    9     7 103 FIFO     Kthread --- Waiting  Semaphore 00000000 008176 002336  28.5%  BT RX  0xf11460f0
   10     7 105 FIFO     Kthread --- Waiting  Semaphore 00000000 001072 001052  98.1%! BT ECC  0xf11460f0
   11     7 103 FIFO     Kthread --- Waiting  Signal    00000000 008176 002236  27.3%  BT Driver  0xf11460f0
   12     3 102 FIFO     Kthread --- Waiting  Semaphore 00000000 008176 001516  18.5%  BT Mesh adv  0xf1104710
```

Reply:
Ok. thank you for the explanation.